### PR TITLE
justedennnn : apply to be istio member

### PR DESCRIPTION
**Pull request by justedennnnn**
The most common reason to send a PR to this repository is to join the Istio org.  If you are joining, please fill out this template.  If you are not, please **delete all the below**.

- [x] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [x] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [x] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [x] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [x] I have joined [Istio's Slack workspace](https://slack.istio.io)

List your company name, or indicate Individual if you're not affiliated with a company:
Daocloud
 
Provide a link to at least one PR that you have successfully pushed to one of the Istio repos:

[#12843](https://github.com/istio/istio.io/pull/12843)
[#11988](https://github.com/istio/istio.io/pull/11988)

### CODE REVIEW

**Comment by zirain** [comment]
please link at least one merged PR, thanks!

File: org/members.yaml [hunk 1]
```diff
@@ -432,3 +432,4 @@ members:
 - zqzzq
 - zufardhiyaulhaq
 - zyfjeff
+- justedennnnn
```
Comment (zirain) [inline org/members.yaml:435]: please sort in alphabetical order

**Comment by windsonsea** [comment]
Welcome, please list [what you have merged in istio.io](https://github.com/istio/istio.io/pulls?q=is%3Apr+author%3A%22justedennnnn+%22+is%3Aclosed), like:
- [istio.io#12843](https://github.com/istio/istio.io/pull/12843)
- [istio.io#11988](https://github.com/istio/istio.io/pull/11988)

**Review by zirain — APPROVED** [review]
Thanks, and Welcome!

**Pull request merged**